com.unity.netcode.gameobjects/Components/NetworkTransform.cs

```diff
@@ -844,35 +946,82 @@ private void Awake()
             }
         }
 
+        /// <summary>
```

Comment:
I usually try to keep most of my properties at the top of the class together, dunno if there's a convention for this in the SDK team

Reply:
Yeah, I might be removing these... with the newer fix in place (improved over the Bitset branch), I am backing out other changes (additions) that are no longer needed now that we are additively updates the state during the current network tick up until the state is sent where it resets to prepare for capturing the transform state for the next tick.

If I end up keep these then they will definitely get migrated up to the rest of the properties. 👍